pkg/client/dtclient/client.go

```diff
@@ -126,6 +125,8 @@ type SettingsClient interface {
 	// ListSchemas returns all schemas that the Dynatrace environment reports
 	ListSchemas() (SchemaList, error)
 
+	FetchSchemasConstraints(schemaID string) ([][]string, error)
```

Comment:
```suggestion
	FetchSchemasConstraints(schemaID string) (SchemaConstraints, error)
```

and somewhere 
```
type SchemaConstraints [][]string
```

Please also add documentation to the method & SchemaConstraints type

Reply:
It will be improved in a separate commit...